testing/helpers/checkStyleHelper.js

```diff
@@ -0,0 +1,53 @@
+import devices from "core/devices";
+import browser from "core/utils/browser";
```

Comment:
import { webkit } from "core/utils/browser";

Reply:
The 'browser.' prefix make it easier to understand what the 'webkit' variable value means:
'if(browser.webkit) {' vs 'if(webkit) {'
The 'import { webkit }' and 'if(webkit) {...' approaches are not used in our code so I don't want to introduce them for the first time if I am not sure that they are better.

Reply:
I think everyone is very clear that `webkit` means in the test`s context :). Our code is not ideal, if we can do it better - we should.

Reply:
I don't see any advantage in using the 'if(webkit)' template instead of the widely used 'if(browser.webkit)' template. For a single place among a lot of others I see disadvantages: it can make it harder to review all the 'browser.webkit' occurrences and can add some time to make sure that the 'webkit' value means the usual 'browser.webkit' value.
We should use the same condition syntax in this file as in other files.

Reply:
It doesn't matter to me, let it be `browser.webkit`